finder_server.h

```diff
@@ -15,36 +15,36 @@
 #include "supplyfinder.grpc.pb.h"
 #endif
 
+struct Comp {
+  inline bool operator() (const supplyfinder::ShopInfo& lhs, const supplyfinder::ShopInfo& rhs) {
+    return (lhs.inventory().price() < rhs.inventory().price());
+  }
+};
+
 class VendorClient {
   /*
    * VendorClient talks to vendor servers. Created when the Finder receives 
    * VendorInfo from Supplier Server
    */
   public:
-  VendorClient(std::shared_ptr<grpc::Channel> vendor_channel):
-  vendor_stub_(supplyfinder::Vendor::NewStub(vendor_channel)) {}
-  
-  supplyfinder::InventoryInfo InquireInventoryInfo (uint32_t);
+    VendorClient(std::shared_ptr<grpc::Channel> vendor_channel):
```

Comment:
Another style nit here. I didn't realize that these two lines were part of the constructor.  Can you put the definition of the constructor in the cc file?